_tests/browser_tests/environment.js

```diff
@@ -0,0 +1,28 @@
+// Common configuration files with defaults plus overrides from environment vars
+var webServerDefaultPort = 7000;
+
+module.exports = {
+  // The address of a running selenium server.
+  seleniumAddress:
+    (process.env.SELENIUM_URL || 'http://localhost:4444/wd/hub'),
+
+  // Capabilities to be passed to the webdriver instance.
+  capabilities: {
+    'browserName':
+        (process.env.TEST_BROWSER_NAME || 'chrome'),
```

Comment:
This file is great idea. I am not sure we need both multi-capabilities and capabilities. We should probably define the environments and environment specific variables here.